File: IfStatements/HowOldAreYouSpecifically.py
name = raw_input("Hey, what's your name? ")
print ""
print "Ok, %r, how old are you? " % name
age = raw_input()
print ""
if age < "16":
    print "You can't drive, %s." % name
elif age <"18":
    print "You can drive but not vote, %s" % name
elif age < "25":
    print "You can vote but not rent a car, %s." % name
else:
    print "You can do anything that's legal, %s." % name

File: IfStatements/HowOldAreYouSpecifically.java
import java.util.Scanner;
public class HowOldAreYouSpecifically {
	public static void main(String[] args){

	    String name;
	    int age;
	    Scanner keyboard = new Scanner(System.in);

	    System.out.print("Hello, What's your name? (Sorry I keep forgetting.) ");
	    name = keyboard.next();
	    System.out.println(" ");
	    System.out.print("Ok, " + name + " , how old are you? ");
	    age = keyboard.nextInt();
	    
	    if (age < 16) {
	    	System.out.println("You can't drive, " + name + " .");
	    }
	    else if (age < 17){
	    	System.out.println("You can drive but not vote, " + name + " .");
	    }
	    else if (age < 25){
	    	System.out.println("You can vote but not rent a car, " + name + " .");
	    }
	    else {
	    	System.out.println("You can do pretty much anything legal, " + name + " .");
	    }
	  }
}
